### ISSUE TITLE
Document your API with Swagger

### ISSUE THREAD

**Issue by galadril** [apption-labs/meater-cloud-public-rest-api#13]
Please document your API with a Swagger definition (OpenAPI 3). This would benefit yourself (the option to generate developer portals), but also your clients. You would be able to generate client code from swagger definitions.

You could use a tool like SwaggerHub from Smartbear for example:
https://swagger.io/docs/specification/2-0/what-is-swagger/

**Comment by galadril** [comment]
You can also use Github Pages to host your swagger api doc:
https://medium.com/@crshnburn/embed-swagger-ui-in-github-pages-9c77a86b2fd2

**Comment by nojacko** [comment]
That's another good one. I like Swagger. But since this API's scope is pretty small right now the README works OK (could be better). When we add more features to the API I'll definitely look to implement this.

**Comment by galadril** [comment]
True. I'm just giving some enhancement ideas as Im currently working on integrating Meater in my SloGrill BBQ Logbook recording app 🗡️ . Thanks for the api.

![Naamloos-2](https://user-images.githubusercontent.com/14561640/106913467-4def7180-6704-11eb-960a-d373f9646d47.jpg)

**Comment by nojacko** [comment]
That's cool! Our customers over in the MEAT 'n' Greet Facebook group would probably love that!

https://facebook.com/groups/meater/
